.storybook/stories/Sky.stories.js

```diff
@@ -8,13 +10,16 @@ import { Plane } from '../../src/shapes'
 export default {
   title: 'Abstractions/Sky',
   component: Sky,
-  decorators: [(storyFn) => <Setup> {storyFn()}</Setup>],
+  decorators: [withKnobs, (storyFn) => <Setup> {storyFn()}</Setup>],
 }
 
 function SkyScene() {
   return (
     <>
-      <Sky sunPosition={[0, Math.PI, -1]} />
+      <Sky
+        distance={number('distance', 450000)}
+        sunPosition={[number('sunPosX', 0), number('sunPosY', Math.PI), number('sunPosZ', -1)]}
```

Comment:
Can you use range for the inputs? See the story for MeshWobbleMaterial

Reply:
What could be a reasonable range for a position vector?

Reply:
https://github.com/mrdoob/three.js/blob/master/examples/webgl_shaders_sky.html#L61

In THREE.js official samples is calculated from euler angles, for example, but I don't think this can be a good option, since they are not the real props.